### PR TITLE
Update/Improve CMake build

### PR DESCRIPTION
**Pull request by tim-gromeyer**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -62,5 +70,29 @@ if(QMARKDOWNTEXTEDIT_EXE)
     )
 
     add_executable(qmarkdowntextedit-exe ${SOURCE_FILES})
-    target_link_libraries(qmarkdowntextedit-exe Qt${QT_VERSION_MAJOR}::Widgets ${INTL_LDFLAGS} qmarkdowntextedit)
+    set_target_properties(qmarkdowntextedit-exe PROPERTIES OUTPUT_NAME "qmarkdowntextedit")
```
Comment (tim-gromeyer) [inline CMakeLists.txt:73]: This will create `qmarkdowntextedit.exe` instead of `qmarkdowntextedit-exe.exe`

**Comment by pbek** [comment]
@Waqar144, objections?

**Comment by Waqar144** [comment]
Looks good! Thank you

**Comment by pbek** [comment]
Thank you!

**Pull request merged**